frontend/src/features/notifications/components/NotificationInformationModal.tsx

```diff
@@ -0,0 +1,381 @@
+            </Grid>
+            <Grid display="grid" item xs={6}>
+                <FormLabel>Leading Root Cause</FormLabel>
+                {LEADING_ROOT_CAUSE.find((dt) => dt.key === demandCapacityNotification.leadingRootCause)?.value}
+            </Grid>
+            <Grid display="grid" item xs={6}>
+                <FormLabel>Status</FormLabel>
+                {STATUS.find((dt) => dt.key === demandCapacityNotification.status)?.value}
+            </Grid>
+            <Grid display="grid" item xs={6}>
+                <FormLabel>Effect</FormLabel>
+                {EFFECTS.find((dt) => dt.key === demandCapacityNotification.effect)?.value}
+            </Grid>
+            <Grid display="grid" item xs={6}>
+                <FormLabel>Start Date of Effect</FormLabel>
+                {new Date(demandCapacityNotification.startDateOfEffect).toLocaleString()}
+            </Grid>
+            <Grid display="grid" item xs={6}>
+                <FormLabel>Expexted End Date of Effect</FormLabel>
+                {new Date(demandCapacityNotification.expectedEndDateOfEffect).toLocaleString()}
+            </Grid>
+            <Grid display="grid" item xs={12}>
+                <FormLabel>Affected Sites Sender</FormLabel>
+                {demandCapacityNotification.affectedSitesBpnsSender && demandCapacityNotification.affectedSitesBpnsSender.length > 0 ? demandCapacityNotification.affectedSitesBpnsSender.join(", ") : 'None'}
+            </Grid>
+            <Grid display="grid" item xs={12}>
+                <FormLabel>Affected Sites Recipient</FormLabel>
+                {demandCapacityNotification.affectedSitesBpnsRecipient && demandCapacityNotification.affectedSitesBpnsRecipient.length > 0 ? demandCapacityNotification.affectedSitesBpnsRecipient.join(", ") : 'None'}
+            </Grid>
+            <Grid display="grid" item xs={12}>
+                <FormLabel>Affected Material Numbers</FormLabel>
+                {demandCapacityNotification.affectedMaterialNumbers && demandCapacityNotification.affectedMaterialNumbers.length > 0 ? demandCapacityNotification.affectedMaterialNumbers.join(", ") : 'None'}
+            </Grid>
+        </Grid>
+    </Stack>
+}
+
+export const DemandCapacityNotificationInformationModal = ({ open, demandCapacityNotification, onClose, onSave }: DemandCapacityNotificationInformationModalProps) => {
+    const [temporaryDemandCapacityNotification, setTemporaryDemandCapacityNotification] = useState<Partial<DemandCapacityNotification>>({});
+    const { partners } = useAllPartners();
+    const { partnerMaterials } = usePartnerMaterials(temporaryDemandCapacityNotification.partnerBpnl ?? `BPNL`);
+
+    const [notifications, setNotifications] = useState<Notification[]>([]);
+    const [formError, setFormError] = useState(false);
+
+    const { sites } = useSites();
+
+    useEffect(() => {
+        setTemporaryDemandCapacityNotification(prevState => ({
```

Comment:
Does this make sense? 
- Create notification > save
- Create notification > same notification is defaulted > save -> no update but creation of a new

At least defaulting looks a little bit strange + doesn't work when selecting a notification and then clicking on send